src/util/http/MediaTypes.cpp

```diff
@@ -227,11 +178,10 @@ std::optional<MediaType> getMediaTypeFromAcceptHeader(
 }
 
 // ______________________________________________________________________
-std::string getErrorMessageForSupportedMediaTypes(
-    const std::vector<MediaType>& supportedMediaTypes) {
+std::string getErrorMessageForSupportedMediaTypes() {
   // TODO<C++23> Refactor this using `join_view` and the formatting of ranges.
   std::vector<std::string> asString;
-  for (const auto& type : supportedMediaTypes) {
+  for (const auto& type : SUPPORTED_MEDIA_TYPES) {
     asString.push_back(toString(type));
   }
```

Comment:
While you are at it, this TODO can be addressed as we have `ad_utility::lazyStrJoin` (in `util/Algorithm.h`)
(sprinkle in a `views::transform` for the `toString`.